fili-core/src/main/java/com/yahoo/bard/webservice/web/FilteredSketchMetricsHelper.java

```diff
@@ -243,7 +243,7 @@ public TemplateDruidQuery updateTemplateDruidQuery(
             JsonNode metricFilterObject,
             DimensionDictionary dimensionDictionary,
             LogicalTable table,
-            DataApiRequest apiRequest
+            DataApiRequestImpl apiRequest
```

Comment:
This change doesn't look right. Shouldn't you be passing the interface here instead of the class? If not, it deserves a specific comment saying why the interface was not sufficient.